kuma/api/v1/plus/notifications.py

```diff
@@ -397,27 +397,24 @@ def update(request, body: UpdateNotificationSchema):
     return 200, True
 
 
-class CreatePRNotificationSchema(Schema):
+class ContentUpdateNotificationSchema(Schema):
     raw_url: str = Field(..., alias="page")
-    repo: str = Field(..., alias="repo")
-    pr: int
+    pr_url: str = Field(..., alias="pr")
 
 
-@admin_router.post("/create/pr/", response={200: Ok, 400: NotOk, 401: NotOk})
-def create_pr(request, body: CreatePRNotificationSchema):
-    url = DocumentURL.normalize_uri(body.raw_url)
-    watchers = Watch.objects.filter(url=url)
-    if not watchers:
-        return 400, {"error": "No watchers found"}
-
-    content = f"Page updated (see PR!{body.repo.strip('/')}!{body.pr}!!)"
-    notification_data, _ = NotificationData.objects.get_or_create(
-        text=content, title=watchers[0].title, type="content", page_url=body.raw_url
-    )
-
-    for watcher in watchers:
-        # considering the possibility of multiple pages existing for the same path
-        for user in watcher.users.all():
-            Notification.objects.create(notification=notification_data, user=user)
+@admin_router.post("/update/content/", response={200: Ok, 400: NotOk, 401: NotOk})
```

Comment:
Could this get a test?

Reply:
Added in 8f35a2f40f3942051bccdc6180eac06b6bed6dcf, thanks for helping me out with this.